mkdocs.yml

```
@@ -28,8 +28,10 @@ plugins:
 
 extra:
   url: &url !!python/object/apply:os.getenv ["DEPLOY_URL", "http://localhost:8000"]
-  assets: &assets !!python/object/apply:os.path.join [*url, "assets"]
-  resources: !!python/object/apply:os.path.join [*assets, "resources"]
+  _url_path: &_url_path !!python/object/apply:urllib.parse.urljoin [*url, "./"]
+  assets: &assets !!python/object/apply:urllib.parse.urljoin [*_url_path, "assets"]
+  _assets_path: &_assets_path !!python/object/apply:urllib.parse.urljoin [*_url_path, "assets/"]
+  resources: !!python/object/apply:urllib.parse.urljoin [*_assets_path, "resources"]
```

Could you please change the variables to:
```suggestion
extra:
  url: &url !!python/object/apply:os.getenv ["DEPLOY_URL", "http://localhost:8000"]
  assets: !!python/object/apply:urllib.parse.urljoin [*url, "/assets"]
  resources: !!python/object/apply:urllib.parse.urljoin [*url, "/assets/resources"]
```
It is easier to read and understand (and I can live without referencing the `assets` variable in the `resources`) :)

---

@steve-todorov I should have added comment on why these were necessary. But I agree your sugg